audience_manager/api.rst

```diff
@@ -0,0 +1,213 @@
+
+    :param string error: Error code. Example::
+
+        'server_error'
+
+Check audience
+``````````````
+Checks if user is part of specific audience.
+
+Code::
+
+    ppms.am.api('checkAudience', audience_id, onFulfilled, onRejected);
+
+.. data:: audience_id
+
+    ID of checked audience. Example::
+
+        '52073260-5861-4a56-be5e-6628794722ee'
+
+.. function:: onFulfilled(in_audience)
+
+    Function executed on success.
+
+    :param boolean in_audience: True when user is part of audience, false otherwise.
+
+.. function:: onRejected(error)
+
+    Function executed on error.
+
+    :param string error: Error code. Example::
+
+        'server_error'
+
+Get attributes
+``````````````
+Gets user attributes.
+
+.. warning::
+
+    Only white-listed attributes are available this way.
+
+Code::
+
+    ppms.am.api('getAttributes', onFulfilled, onRejected);
+
+.. function:: onFulfilled(attributes)
+
+    Function executed on success.
+
+    :param Object<string,Object<string,string>> attributes: Object containing all white-listed attributes. It's divided
+        on by categories. 'analytics' object contain analytical data about the user (e.g. browser name and version,
+        country, etc.). 'attributes' object contain data from other sources (e.g. Audience Manager Form Tracker, data
+        merged via CSV import etc.). Example::
+
+            {
+                "analytics": {
+                    "browser_name": "chrome",
+                    "country": "us",
+                },
+                "attributes": {
+                    "first_name": "James",
+                    "last_name": "Bond"
+                }
+            }
+
+.. function:: onRejected(error)
+
+    Function executed on error.
+
+    :param string error: Error code. Example::
+
+        'server_error'
+
+Update attributes
+`````````````````
+Updates user attributes.
+
+Code::
+
+    ppms.am.api('updateAttributes', attributes, options);
+
+.. data:: attributes
+
+    Object containing attributes to update. Every key and value should be a string. Example::
+
+        {
+            "favourite_color": "black",
+            "drink: "Martini"
+        }
+
+.. data:: options
+
+    **Optional** object that lets execute command with extra parameters. Example::
+
+        {
+            "user_id": user_id,
+            "device_id": device_id,
+            "email": email,
+            "onFulfilled": onFulfilled,
+            "onRejected": onRejected
+        }
+
+    .. attribute:: user_id
+
+        If application lets user login this option will let Audience Managed better identify user across sessions. It's
+        value should be unique user identifier provided by the App. It may be user handle or internal user ID. Example::
+
+            "jbond"
+
+    .. attribute:: device_id
+
+        If application has access to device id this option will let Audience Managed better identify user across
+        sessions. Example::
+
+            "007"
```

Comment:
In my opinion, the example should be similar to real device ID. For sure none of the device IDs consists of 3 characters.